luaui/Widgets/gui_gridmenu.lua

```diff
@@ -857,6 +857,16 @@ function prevPageHandler()
 	return true
 end
 
+function goToCategoriesHandler()
+	if not currentBuildCategory then
```

Comment:
Ideally check that you have a `selectedBuilder`, just for sanity. E.g. we messed up selection from builder to factory and didn't clean up the variables well.

Not a blocker.